public/docs/dart/latest/tutorial/toh-pt5.jade

```diff
@@ -1,702 +1,168 @@
-+makeExample('toh-5/dart/lib/dashboard_component_1.dart',null, 'lib/dashboard_component.dart (v1)')(format=".")
-:marked
-  We’ll come back and make it more useful later.
+    The `AppComponent` doesn't have a router yet. We'll use the `@RouteConfig`
+    annotation to simultaneously:
 
-  ### Configure the dashboard route
-  Go back to `app_component.dart` and teach it to navigate to the dashboard.
+    - Assign a router to the component and
+    - Configure that router with *routes*
 
-  Import the `DashboardComponent` so we can reference it in the dashboard route definition.
-
-  Add the following `'Dashboard'` route definition to the `@RouteConfig` list of definitions.
-+makeExample('toh-5/dart/lib/app_component.dart','dashboard-route', 'lib/app_component.dart (Dashboard route)')(format=".")
-.l-sub-section
+block routerLink
   :marked
-    **useAsDefault**
-
-    We want the app to show the dashboard when it starts and
-    we want to see a nice URL in the browser address bar that says `/dashboard`.
-    Remember that the browser launches with `/` in the address bar.
-    We don't have a route for that path and we'd rather not create one.
-
-    Fortunately we can add the `useAsDefault: true` property to the *route definition* and the
+    Notice the `[routerLink]` binding in the anchor tag.
+    We bind the `RouterLink` directive (another of the `ROUTER_DIRECTIVES`) to a list
+    that tells the router where to navigate when the user clicks the link.
+
+    We define a *routing instruction* with a *link parameters list*.
+    The list only has one element in our little sample, the quoted ***name* of the route** to follow.
+    Looking back at the route configuration, we confirm that `'Heroes'` is the name of the route to the `HeroesComponent`.
+  .l-sub-section
+    :marked
+      Learn about the *link parameters list*
+      in the [Routing](../guide/router.html#link-parameters-array) chapter.
+
+block redirect-vs-use-as-default
+  :marked
+    We don't need a route definition for that. Instead,
+    we add `useAsDefault: true` to the dashboard *route definition* and the
     router will display the dashboard when the browser URL doesn't match an existing route.
-:marked
-  Finally, add a dashboard navigation link to the template, just above the *Heroes* link.
 
-+makeExample('toh-5/dart/lib/app_component.dart','template', 'lib/app_component.dart (template)')(format=".")
-.l-sub-section
+block templateUrl-path-resolution
   :marked
-    We nestled the two links within `<nav>` tags.
-    They don't do anything yet but they'll be convenient when we style the links a little later in the chapter.
-:marked
-  Refresh the browser. The app displays the dashboard and
-  we can navigate between the dashboard and the heroes.
+    The given path must resolve to the named [asset][] within the
+    application's package. Assests from other packages can also be accessed
```

Comment:
This made me wonder whether the path is relative to this file or to the `lib/` directory. It'd be good to state that explicitly.